l10n_it_website_portal_fiscalcode/controllers/main.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Simone Rubino
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import ValidationError
+from odoo.http import request
+
+from odoo.addons.portal.controllers.portal import CustomerPortal
+
+CustomerPortal.OPTIONAL_BILLING_FIELDS.extend(["fiscalcode"])
+
+
+class WebsitePortalFiscalCode(CustomerPortal):
+    def details_form_validate(self, data):
+        error, error_message = super(
+            WebsitePortalFiscalCode, self
+        ).details_form_validate(data)
```

Comment:
la firma di details_form_validate è cambiata:

```suggestion
    def details_form_validate(self, data, partner_creation=False):
        error, error_message = super(
            WebsitePortalFiscalCode, self
        ).details_form_validate(data, partner_creation)
```